tests/README.md

```diff
@@ -47,6 +47,10 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 bash tests/docker/ducker-ak up -j 'openjdk:11'; tests/docker/run_tests.sh
 ```
+* Clean Docker Image
```

Comment:
`remove ducker-ak nodes`

Reply:
Is "remove ducker-ak images" better?
If you feel the same way, I'll change it to this

Reply:
`Container` is good to me

Reply:
I update it,thank you